bat/compute.go

```diff
@@ -103,15 +103,6 @@ type NodeStatus struct {
 	Hostname              string    `json:"hostname"`
 }
 
```

Comment:
Don't we need to delete RunCIAOCLI and friends or are they still being used somewhere?